bubblewrap.c

```diff
@@ -2081,8 +2081,23 @@ write_namespace_ids (int   fd,
   if (pid_fd < 0)
     die_with_error ("open /proc/%s failed", dir);
 
+  if (opt_unshare_user)
+    write_id_for_namespace (fd, pid_fd, "user", in_json);
+
   if (opt_unshare_pid)
     write_id_for_namespace (fd, pid_fd, "pid", in_json);
+
+  if (opt_unshare_ipc)
+    write_id_for_namespace (fd, pid_fd, "ipc", in_json);
+
+  if (opt_unshare_net)
+    write_id_for_namespace (fd, pid_fd, "net", in_json);
+
+  if (opt_unshare_uts)
+    write_id_for_namespace (fd, pid_fd, "uts", in_json);
+
+  if (opt_unshare_cgroup)
```

Comment:
This doesn't propertly handle opt_unshare_cgroup_try. Also, if that is true it can't fail if the file is not there. Maybe we just need to set the opt_unshare_cgroup to true in the _try case when it is used, like we do for user_try.

Reply:
Let me do the latter.

Reply:
Done